user-manual/administer/settings.rst

```diff
@@ -2215,6 +2216,95 @@ To save any modifications, click the "Save" button located below the
+
+.. IMPORTANT::
+
+   Both the **arStorageServicePlugin** and **arRestApiPlugin** must be enabled for this "Storage service" option to appear on the Settings menu and for the Archivematica integration to work. See :ref:`Manage Plugins <manage-plugins>`.
+
+1. Fill in the appropriate value for the address of the Storage Service API 
+   used by the Archivematica instance. This is typically located at port 8000. 
+   Don't forget to add the api version.
+2. Log into the Storage Service with an authorized account. Go to 
+   **Administration > Users** and  click the "Edit" button for an active 
+   account. Note the "Username" and copy the "Api key" at the bottom of the page.
+3. Enter these values in the "Storage Service username" field and "Storage 
+   Service API key" field on the AtoM Storage Service settings form.
+4. Toggle the "Enable AIP download" option to "Enabled". Press the "Save" 
+   button.
+
+.. image:: images/storage-service-settings.*
+   :align: center
+   :width: 90%
+   :alt: Storage service settings
+
+.. NOTE::
+  
+   By default this will only enable Archivematica File and AIP download for users that are members of the "Adminstrator" group. 
+
+5. Browse to a digital object that was created using the DIP Upload feature. 
+   Download links are located to the right of the File and AIP UUIDs.
+
+.. image:: images/digital-object-preservation-copies-with-access.*
+   :align: center
+   :width: 90%
+   :alt: Digital object preservation copies, with download access 
+
+.. NOTE::
+  
+   When these links are clicked a new tab will temporarily open and will 
+   remain open until the Storage Service responds to the request. Once the 
+   file download begins, the download tab will disappear to be replaced with 
+   the download status icon for your browser. The files are saved with the 
+   name sent in the Storage Service response. If there is an error with the 
+   call to the Storage Service, the error status page will be displayed in 
+   this tab.
+
+6. To enable this feature for other user groups you must update the 
+   security.yml file in the arStorageServicePlugin folder. Starting from the root folder of the AtoM application, e.g.
+
+    .. code-block:: bash 
+       
+       /usr/share/nginx/atom
+
+    it is found at:
+
+    .. code-block:: bash 
+       
+       plugins/arStorageServicePlugin/modules/arStorageService/config/security.yml
+
+7. Add user groups to the credentials list. The default content is:
+
+    .. code-block:: bash 
+       
+       all:
+         is_secure: true
+         credentials: administrator
+
+    Example content for adding the editor and translator group:
+
+    .. code-block:: bash 
+       
+       all:
+         is_secure: true
+         credentials: [[ administrator, editor, translator ]]
+
+
+  8. After making changes to the security.yml file, refresh the application 
```

Comment:
Does PHP-FPM need to be restarted too? I know this is the case for most other config file changes. Might want to add it just to be safe, but you're the one who has tested, so I'll leave it up to you!

Because FPM restart commands tend to change based on PHP versions, I typically link to the docs where we show how to restart it (in Troubleshooting), rather than add it per-page, so there's only one place to update in future docs. 

For reference, if you do want to add it: 

```
:ref:`troubleshooting-restart-php-fpm`

Reply:
Steve and Evelyn did QA for this. I only volunteered to do docs because it's closely tied into DO display. So I was going off the instructions that Steve left. To be sure, I've just tested this. Interestingly only the `php symfony cc` is needed to make it work. No PHP-FPM restart  needed :-)

Reply:
ok :)